contracts/message-bus/MessageOutbox.sol

```diff
@@ -72,13 +72,17 @@ contract MessageOutbox is MessageBox {
     /**
      * @notice Setup message outbox.
      *
-     * @dev Function requires:
-     *          - outboundChannelIdentifier must be zero
-     *          - metachainId must not be zero
-     *          - messageInbox address must not be zero
-     *
      * @param _metachainId Metachain identifier.
      * @param _messageInbox MessageInbox contract address.
+     *
+     * \pre Setup can only be called once.
```

Comment:
The above mentioned points are also applicable here for pre and post conditions.

Reply:
Fixed